.github/workflows/deploy.yml

```
@@ -11,10 +11,10 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
```

OPTIONAL:
```suggestion
    - uses: actions/checkout@v3
```
https://github.com/actions/checkout/releases